### PR TITLE
chore(litmus-portal): Tune workflow screen and workflow page integration.

### LINKED ISSUE #1579: Workflows- Tune workflow screen Dev.
Development of 'Tune Workflow' screen for Litmus Portal.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

**What this PR does / why we need it**:

Added Yaml Editor for Workflow CRDs, tune workflow screen and workflow page integration.

**Which issue this PR fixes**

fixes #1579 #1575 

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [x] Have you added debug messages where necessary?
* [x] Have you added task comments where necessary?
* [x] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any?
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [x] Do the artifacts follow the appropriate directory structure?
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks?
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted?

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
@rajdas98 @Jonsy13 @oumkale @arkajyotiMukherjee @S-ayanide @gdsoumya @Saranya-jena @amityt PTAL.

File: litmus-portal/frontend/src/components/TuneWorkflow/styles.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: 'rgba(255, 255, 255, 0.6)',
+    height: '100%',
+    width: '61.25rem',
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/components/TuneWorkflow/styles.ts:7]: This is a very minor change to make it 80% of the whole screen width(covered in #1808 ) to fit the design requirement, its not an urgent need but could be modified later.

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/TuneWorkflow/styles.ts]: Done.

**Comment by ishangupta-ds** [comment]
@S-ayanide, @rajdas98  and @oumkale PTAL.

**Comment by ishangupta-ds** [comment]
@rajdas98 @oumkale @S-ayanide @amityt @Jonsy13 PTAL, resolved some merge conflicts.

File: litmus-portal/frontend/package.json [hunk 2]
```diff
@@ -20,17 +20,22 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/pickers": "^3.2.10",
+    "@types/js-yaml": "^3.12.5",
```
Comment (S-ayanide) [inline litmus-portal/frontend/package.json:23]: @types should be a dev dependency

Comment (ishangupta-ds) [reply litmus-portal/frontend/package.json]: Done.

**Comment by ishangupta-ds** [comment]
@S-ayanide @Saranya-jena @Jonsy13 @oumkale @rajdas98 PTAL.

**Pull request merged**